test/test_util.hpp

```diff
@@ -25,9 +25,9 @@ namespace llarp
 
     struct FileGuard
     {
-      const fs::path &p;
+      const fs::path p;
```

Comment:
Note: I'm assuming this was an optimization, which isn't really necessary for unit testing. It was also restrictive in the way it can be used.